gym_roboy/envs/msj_env.py

```diff
@@ -45,6 +47,7 @@ def __init__(self, ros_proxy: MsjROSProxy = MsjROSBridgeProxy(),
             joint_angle=-self._JOINT_ANGLE_BOUNDS, joint_vel=-self._JOINT_VEL_BOUNDS, is_feasible=False)
         self._joint_vel_penalty = joint_vel_penalty
         self._is_tendon_vel_dependent_on_distance = is_tendon_vel_dependent_on_distance
+        self._is_agent_getting_bonus_for_done = is_agent_getting_bonus_for_done
```

Comment:
in case we introduce a maximum episode length, then the robot can be done without actually reaching the goal. In those cases we would not give a bonus. Maybe the correct variable name is `is_agent_getting_bonus_for_reaching_goal`

Reply:
`is_agent_getting_bonus_for_reaching_goal` is indead a better name.
I added your suggestion.